robustocs/loaders.py

```diff
@@ -149,7 +149,7 @@ def load_sparse_symmetric_matrix(filename: str, dimension: int, nnz: int,
 def load_sexes(filename: str, dimension: int) -> tuple[
     npt.NDArray[np.unsignedinteger],
     npt.NDArray[np.unsignedinteger],
-    npt.NDArray[np.str]
+    npt.NDArray[np.str_]
```

Comment:
This is specifically only a problem in versions of NumPy released _after_ the one packaged with my Ubuntu which is why I didn't catch this locally :woman_facepalming: